Sources/BraveWallet/Crypto/Stores/NetworkSelectionStore.swift

```diff
@@ -11,12 +11,14 @@ struct NetworkPresentation: Equatable, Hashable, Identifiable {
   enum Network: Equatable, Hashable {
     case allNetworks
     case network(BraveWallet.NetworkInfo)
+    case networkSelection(BraveWallet.NetworkInfo?)
```

Comment:
Is this needed? Can't we just re-use `network(BraveWallet.NetworkInfo)` case and avoid optional entirely? `networkSelectionInForm: BraveWallet.NetworkInfo?` is optional because the initial value is not set, but after that there is no way to set to nil so I'm not seeing why this new `NetworkPresentation.Network` case is needed 🤔

Reply:
addressed in [15f37fc](https://github.com/brave/brave-ios/pull/6346/commits/15f37fc85e89719dba2dd37a433a845c1ba47f8f)